util-moduleloader/pom.xml

```diff
@@ -35,6 +35,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>caf-logging</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
Remove this.  Only the outermost component should specify the logging framework.

Reply:
removed